spec/features/upload_download_mous_spec.rb

```diff
@@ -23,14 +23,24 @@
     end
 
     context 'uploading the signed mou' do
-      it 'can upload and download a version of the mou' do
+      before do
         visit mou_index_path
-
         attach_file("signed_mou", Rails.root + "spec/fixtures/mou.pdf")
         click_on 'Upload'
+      end
 
+      it 'can upload and download a version of the mou' do
         expect(page).to have_content("MOU uploaded successfully.")
       end
+
+      it 'allows the user to see when the mou was uploaded' do
+        expect(page).to have_content("A signed MoU was uploaded on")
+      end
+
+      it 'allows the user to click the link and download their MoU' do
+        expect(page).to have_content("download and view the document.")
+        click_on 'download and view the document.'
```

Comment:
Is this click_on asserting that the button is on the page?
I would usually expect an action like this with an assertion at the end.

Reply:
Yeah I don't need that click really the line above ensures its on the page anyway